internal/pkg/consensus/signing.go

```diff
@@ -0,0 +1,59 @@
+package consensus
```

Comment:
nit: a slight name change would be nice as this doesn't do any signing, only signature validation

Reply:
Done, also moved out of consensus package to state package (alongside view).